kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/ConfirmationPaneViewModel.java

```diff
@@ -0,0 +1,30 @@
+package dev.ikm.komet.kview.mvvm.viewmodel;
+
+import org.carlfx.cognitive.viewmodel.SimpleViewModel;
+
+/**
+ * The view model for the confirmation pane.  This view model manages the text that is displayed
+ * on the confirmation pane, and the notification topic and event that is sent when the close
+ * button is pressed.
+ */
+public class ConfirmationPaneViewModel extends SimpleViewModel {
+
+    /**
+     * The properties of the view model.
+     */
+    public enum ConfirmationPropertyName {
+        CONFIRMATION_TITLE,
+        CONFIRMATION_MESSAGE,
+        CLOSE_CONFIRMATION_PANEL
```

Comment:
Can you add the `WINDOW_TOPIC`? 
**Reason:** Used to emit event (message to close properties bumpout) when 'CLOSE_CONFIRMATION_PANEL' property is `true`.